gradle/libs.versions.toml

```diff
@@ -2,24 +2,19 @@
 
 detektGradlePlugin = "1.23.6"
 junit = "4.13.2"
-kotlinExposed = "1.4.32"
+kotlinExposed = "1.7.21"
 dokka = "1.9.20"
 lint = "30.1.0"
 openTelemetryCore = "1.38.0"
 openTelementrySemConv = "1.25.0-alpha"
-moshi = "1.12.0"
-lifecycle = "2.5.0"
-lifecycle-extensions = "2.0.0"
-mockk = "1.13.11"
-androidx-test = "1.6.1"
-androidx-test-junit = "1.2.1"
-robolectric = "4.13"
-mockwebserver = "4.12.0"
+moshi = "1.14.0"
+lifecycle = "2.8.3"
+lifecycle-extensions = "2.2.0"
 protobuf = "4.27.2"
-profileinstaller = "1.0.0"
-okhttp = "4.9.3"
+profileinstaller = "1.3.1"
+okhttp = "4.11.0"
 firebase = "23.1.0"
-compose = "1.0.5"
+compose = "1.5.4"
```

Comment:
Is it essential to migrate to these deps as part of 1.7, and if so how can we test the changes for code that isn't really covered (such as compose listeners, unless I've missed the test coverage)?

Reply:
I assumed there was test coverage, but let me revert those that don't. I think Lifecycle and OkHttp should be fine, but profile installer and compose may not be fully covered.